File: GTTS/INL.VettingService.Models/CourtesyBatch_Item.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace INL.VettingService.Models
{
    public class CourtesyBatch_Item : ICourtesyBatch_Item
    {
        public long CourtesyBatchID { get; set; }
        public long VettingBatchID { get; set; }
        public int VettingTypeID { get; set; }
        public string VettingType { get; set; }
        public string VettingBatchNotes { get; set; }
        public int? AssignedToAppUserID { get; set; }
        public string AssignedToAppUserName { get; set; }
        public DateTime? ResultsSubmittedDate { get; set; }
        public int? ResultsSubmittedByAppUserID { get; set; }
        public string ResultsSubmittedByAppUserName { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.VettingService.Models/courtesy-batch_item.ts


import { ICourtesyBatch_Item } from './icourtesy-batch_item';

export class CourtesyBatch_Item implements ICourtesyBatch_Item {
  
	public CourtesyBatchID: number = 0;
	public VettingBatchID: number = 0;
	public VettingTypeID: number = 0;
	public VettingType: string = "";
	public VettingBatchNotes: string = "";
	public AssignedToAppUserID?: number;
	public AssignedToAppUserName: string = "";
	public ResultsSubmittedDate?: Date;
	public ResultsSubmittedByAppUserID?: number;
	public ResultsSubmittedByAppUserName: string = "";
  
}


